### ISSUE TITLE
FrontEnd enhancement for Displaying suggested courses in Autocomplete.

### ISSUE THREAD

**Issue by nabeelahmadkh** [UIOWA5830SP19/SPP200#53]
When User is searching for courses he should be able to see 5 courses in autocomplete.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jwryan9** [comment]
Is this dependent on #43?

**Comment by nilbull** [comment]
No, these are two independent components.

**Comment by jwryan9** [comment]
Don't you need the API to return autocomplete suggestions, as described in #43, in order to display the issues to the user?

**Comment by nilbull** [comment]
Yes that's true but we are building those as two are separate components
and backend part is already done.
Regards,
Indranil Guha
------------------------------------------------------------------------------------
Ph.D. Student, Dept of ECE,
University of Iowa
Iowa City, USA, IA-52242
Phone: (+1)319-512-8300
------------------------------------------------------------------------------------
